tests/traits/httpHelpers.php

```diff
@@ -62,7 +62,7 @@ public function stopHttpHalting() {
 	 * Use this at the top of tests that should test behavior for different HTTP responses.
 	 */
 	public function startHttpMocking() {
-		add_filter( 'pre_http_request', [ $this, 'httpMock' ], 1 );
+		add_filter( 'pre_http_request', [ $this, 'httpMock' ], 1, 3 );
```

Comment:
what changed here?

Reply:
Technically nothing but that last arg is the number of params that the hooked function receives.